operator/v1/types_ingress.go

```diff
@@ -161,6 +161,12 @@ type IngressControllerSpec struct {
 	// +optional
 	TLSSecurityProfile *configv1.TLSSecurityProfile `json:"tlsSecurityProfile,omitempty"`
 
+	// clientTLS specifies settings for requesting and verifying client
+	// certificates, which can be used to enable mutual TLS.
```

Comment:
This cannot work for passthrough. Which sorts of routes will this protect?  Is there a customer request to *only* allow such routes?

Reply:
This will protect only edge and reencrypt routes.  It follows from the definition of passthrough that the router does not check client certificates for passthrough routes.  I'll note this in the godoc.